aflr/api_resources/file.py

```diff
@@ -1,19 +1,60 @@
+from posixpath import basename
 import aflr
-from aflr.api_request import APIRequest
+from aflr.helper_classes import ListableResource
+from tkinter import Tk
```

Comment:
what is this for? did not find it in the code

Reply:
deleted